pkg/ebpf/events_processor.go

```diff
@@ -363,6 +364,10 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 	// in case FinitModule and InitModule occurs it means that a kernel module was loaded
 	// and we will want to check if it hooked the syscall table and seq_ops
 	case events.DoInitModule:
+		derive.InvokedContextMutex.Lock()
+		contextID := uint64(len(derive.InvokedContext) + 1)
+		derive.InvokedContext[contextID] = *event
```

Comment:
You save all the event, including the arguments which are irrelevant for deriving the event later on.